packages/svg-sprites/README.md

```diff
@@ -0,0 +1,40 @@
+# Fluent UI System Icons (svg)
+
+This package provides the Fluent UI System Icons as SVG sprites, which allow user agents to request and cache icons at runtime instead of depending on this project monolithically.
+
+## Usage
+
+For security reasons, most user agents require the sprites to be served from the same hostname, protocol and port. During development, requests can be proxied to a specific release on e.g. jsdelivr which you have vetted for integrity, a bucket in the cloud, or served from the service’s file system.
+
+For Storybook, `middleware.js` allows the configuration of `http-proxy-middleware` like so:
+
+```js
+const { createProxyMiddleware } = require('http-proxy-middleware')
+
+const expressMiddleWare = (router) => {
+  router.use(
+    '/sprites',
+    createProxyMiddleware({
+      target: 'http://cdn.jsdelivr.net/npm/fluentui-svg-icon-sprites@1.1.163/',
```

Comment:
Does this reference need to change with the package name change?

Reply:
Ah, indeed, thanks for catching this.